src/google/protobuf/generated_message_tctable_impl.h

```diff
@@ -95,11 +96,11 @@ namespace internal {
 
 #ifndef NDEBUG
 template <size_t align>
-#ifndef _MSC_VER
-[[noreturn]]
-#endif
```

Comment:
Do not remove the `[[noreturn]]` annotation, please. This is part of the signature, and cannot be inferred from the function body.

Reply:
Done. Thank you for the timely review, @mcy